BEIMA.Client/src/pages/Devices/DevicePage.js

```diff
@@ -275,36 +268,50 @@ const DevicePage = () => {
 
     const onImageChange = (event) => {
       if(event.target.files && event.target.files[0]){
-        setImageCopy(URL.createObjectURL(event.target.files[0]))
+        setImageCopyUrl(URL.createObjectURL(event.target.files[0]))
+        setNewImage(event.target.files[0])
       }
     }
 
+    const deleteImage = () => {
+      let delDocs = deletedDocs;
+      delDocs.push(imageCopy.fileUid);
+
+      setDeletedDocs(delDocs);
+      setNewImage(new File([""], ""))
```

Comment:
I would instead set the new image to null, and add the photo's file uid to deletedDocs, or make it its own key in the data object in the update request. Then on the backend depending on the implementation check if we need to delete the photo first then set the new photo if any.

Reply:
I will set this to NULL instead. In the deleteImage function, I do add the uid to the deletedFiles variable and that does work.

- [ ] Change the new image object to null if there is no image

Reply:
Updated to null